index.html

```diff
@@ -420,14 +419,22 @@
           settings</var>' <a>top-level browsing context</a>'s <a>origin</a>
         </li>
       </ol>
+      <dl class='idl' title='dictionary PermissionStorage'>
```

Comment:
I can make it look like:

```
dictionary PermissionStorage {
    // PermissionStorage is just an explanatory device.
    // Instances are never received from or passed to Javascript code.
    required PermissionState state;
};
```

Otherwise, I'm open to suggestions, but I do find that having a concrete list of properties, however expressed, makes it easier to follow specifications.